packages/p2p/src/utils/websocket.js

```diff
@@ -13,15 +13,15 @@ export const init = (websocket, local_currency_decimal_places) => {
 };
 
 const setCurrenciesConfig = website_status_response => {
-    if ('website_status' in website_status_response && ObjectUtils.isEmptyObject(CurrencyUtils.currencies_config)) {
-        CurrencyUtils.setCurrencies(website_status_response.website_status);
+    if ('website_status' in website_status_response && isEmptyObject(getCurrencies())) {
+        setCurrencies(website_status_response.website_status);
     }
 };
 
-const formatMoney = (currency, amount) => CurrencyUtils.formatMoney(currency, amount, true);
+const fmtMoney = (currency, amount) => formatMoney(currency, amount, true);
```

Comment:
i think we can remove this function instead

Reply:
This function was there as a shorthand without having to pass the 3rd argument `true`. The way we change import/export does not seem related to refactoring that. Do you still think we should change it?

Reply:
i see, then thats fine, its just the name made me confused for a while, bcause it means the same thing